src/main/scala/com/timushev/sbt/updates/Reporter.scala

```diff
@@ -47,6 +49,17 @@ object Reporter {
       .filterNot(_._2.isEmpty)
   }
 
+  def overrideDependencies(dependencies: Seq[ModuleID], overrides: Iterable[ModuleID]): Seq[ModuleID] = {
+    def key(id: ModuleID) = (id.organization, id.name)
```

Comment:
This will probably not work correctly in some cross-version cases, like:
```scala
libraryDependencies += "org.typelevel" %% "cats-core" % "1.3.1"
dependencyOverrides += "org.typelevel" % "cats-core_2.12" % "1.4.0"
```

You need to handle overrides to the dependencies after `CrossVersion` is applied.

Reply:
I think that `org.typelevel" % "cats-core_2.12" % "1.4.0"` is incorrect definition of a `ModuleId`. The cross version should be handled by `%%` operator. There shouldn't be any production code to handle incorrect dependency definitions. 

Additionally, sbt does not implement this case either in  "override dependencies" functionality. 

But if you insist, I can implement this.

Reply:
SBT docs say that both declarations are valid, `%%` is just a shortcut. Also, if you create a `build.sbt` file with the two declarations above, it works and resolves correctly to `1.4.0`. So, I think it makes sense to handle both variants.

Reply:
Done